src/cipher/random.ml

```diff
@@ -8,28 +8,22 @@
 (* TODO LATER Is there something with lighter dependencies? Although perhaps
    these are not so bad... *)
 
-let set = ref false
-
 let _initialized = ref None
 
-let initialized () : [ `Initialized ] =
+let initialized () =
   match !_initialized with
   | None -> failwith "Entropy is not initialized."
   | Some v -> Lazy.force v
 
 let initialize f =
-  if not !set
-  then begin
-    ( try
-        let `Initialized = initialized () in ()
-        with
-          Failure _ -> ());
-    set := true ;
-    _initialized := Some (Lazy.from_fun f)
-  end
+  ( try
+      initialized () ;
+    with
+      Failure _ -> ());
```

Comment:
suggestion: remove this `( try .. );` block.

Reply:
Done :+1:.